### DISCUSSION TITLE
Code for generating data from Darcy flow?

### DISCUSSION THREAD

**Discussion by plcrodrigues** [neuraloperator/neuraloperator#387] [Q&A] [answered]
Hi, is there any way of recovering the code used to generate the data in `load_darcy_flow_small` ? Thanks!

**Comment by dhpitt** [comment, marked as answer]
Hi @plcrodrigues , there is some legacy code at an old commit here: https://github.com/neuraloperator/neuraloperator/blob/master/data_generation/darcy/solve_gwf.m

Hope this helps!
